### ISSUE TITLE
Is the Previous Mock PA and lab 7 Mock PA graded? 

### ISSUE THREAD

**Issue by aViolentSalad** [nus-cs2030/2223-s1#252]
Hi im just wondering if the Mock PAs are graded :C

**Comment by Aaronpeh022** [comment]
I think its part of lab assignment which totals 5% meaning if you didnt do it, you just dont get the full 5%.

**Comment by GarlicButterSteak** [comment]
I am under the impression that the previous mock was not, but this current mock is

**Comment by SungMatt** [comment]
@danqiye1 Is there some official answer on this?

**Comment by acekhoon** [comment]
I think this one is one of the assignment that consitutes 5% of individual assignment grade. That's the reason why they don't release model solution for this one.

**Comment by mohddanesh** [comment]
I think this one is since it is called lab7... It is a lab

**Comment by dohaduong** [comment]
My TA confirmed it's a graded lab

**Comment by jingyulim1** [comment]
Hi,

So far, only Practice Assessment#1 for PA#1 (which is non-compulsory and solutions provided) is not graded.

**Comment by ZYN2122** [comment]
Yep Lab 7 should be a graded lab, but not too sure where it falls under the scores
